__tests__/nametag.test.tsx

```diff
@@ -9,25 +9,24 @@ jest.mock('../lib/zoomapi', () => jest.requireActual('../lib/fakezoomapi'));
 
 describe('Page', () => {
   it('renders the heading and input fields', () => {
-    render(<Page />)
 
-    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument()
+    render(<Page />)
     expect(screen.getByText('Full Name')).toBeInTheDocument();
     expect(screen.getByText('Preferred Name')).toBeInTheDocument();
     expect(screen.getAllByText('Select Pronouns')[0]).toBeInTheDocument();
     expect(screen.getByText('Self Disclosure')).toBeInTheDocument();
   })
 
   it('toggles the nametag display on switch change', async () => {
-    render(<Page />)
 
-    let switchInput = screen.getByTestId('colored-switch');
-    expect(switchInput).not.toBeChecked();
+    render(<Page />)
 
-    // fireEvent.change(switchInput, { target: { 'aria-checked': true } });
-    await userEvent.click(switchInput);
+    const element = screen.getByLabelText('Name Tag');
+    expect(element).toBeInTheDocument();
```

Comment:
Can we check more specifically that the switch input and the element labeled "Name Tag" are the same component?